sdk/identity/identity/test/msalTestUtils.ts

```diff
@@ -76,152 +81,202 @@ export const openIdConfigurationResponse: Record<string, string | string[] | boo
   rbac_url: "https://pas.windows.net",
 };
 
-export async function msalNodeTestSetup(
-  testContext?: Mocha.Test,
-  playbackClientId = "azure_client_id"
-): Promise<MsalTestSetupResponse> {
+export async function msalNodeTestSetup(options: {
+  testContext?: Mocha.Test;
+  playbackClientId?: string;
+}): Promise<{
+  cleanup: MsalTestCleanup;
+  recorder: Recorder;
+  sandbox: Sinon.SinonSandbox;
+}>;
+
+export async function msalNodeTestSetup(options: { stubbedToken: AuthenticationResult }): Promise<{
```

Comment:
what about these 2 overloads? Will this be supported by Typescript?

export async function msalNodeTestSetup( testContext?: Mocha.Test,  playbackClientId = "azure_client_id")
export async function msalNodeTestSetup(stubbedToken: AuthenticationResult)

Reply:
Yes, I've just changed the signature to match.